index.html

```diff
@@ -185,7 +185,7 @@ <h1>About Myself</h1>
                 <li>
                     <div class="title border-none">
                         <h5>Languages</h5>
-                        <img src="images/Dropdown-down arrow.svg" alt="dropdown-arrow">
+                        <img src="images/Dropdown-down arrow.svg" alt="dropdown-arrow icon">
                     </div>
```

Comment:
![chanes](https://user-images.githubusercontent.com/93039359/200528504-547ebd6f-554f-41ce-b405-ad43c478242a.PNG)

- [ ] You have changed this line but have not added the link of this file to your pull request summary which is a requirement as you can see in the image above ⬆️

Hint: You can see this [pull request summary](https://github.com/orozCoding/myportfolio/pull/8) to get an idea what we are talking about 😊